components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -283,36 +305,33 @@ subroutine AllocationInit ( bounds, elm_fates)
     if (carbonnitrogen_only) then
         nu_com_phosphatase = .false.
     end if
- 
+
     call get_curr_date(yr, mon, day, sec)
     if (spinup_state == 1 .and. yr .le. nyears_ad_carbon_only) then
       Carbon_only = .true.
      end if
-
-    call cnallocate_carbon_only_set(carbon_only)
-    call cnallocate_carbonnitrogen_only_set(carbonnitrogen_only)
-    call cnallocate_carbonphosphorus_only_set(carbonphosphorus_only)
+     !$acc update device(carbon_only, carbonnitrogen_only,&
+     !$acc carbonphosphorus_only)
+    !!call cnallocate_carbon_only_set(carbon_only)
```

Comment:
remove?

Reply:
removed.  And also went ahead and removed the use statements for the "cnallocate_*" functions